fix: multiple workspaces compatibility list (#2147)
codenvy/che-docs: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc

.Prerequisites

* You have installed an instance of `{prod-short}` by using Helm and the default namespace `{prod-namespace}`.

> * The combination of PVC strategy and access mode matches one of the following cases:

Reviewer: ```suggestion
* The combination of persistent volume claim (PVC) strategy and access mode matches one of the following cases:
```
Consider spelling out this initialism unless you're certain that it is familiar to the reader.

+
.Multiple workspaces compatibility list
|===
| PVC Strategy | `ReadWriteMany` access mode | `ReadWriteOnce` access mode
| `common` (default) | yes | no
| `per-workspace` | yes | yes
| `unique` | yes | yes
|===
+

> ** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes] 

Reviewer 2: ```suggestion
** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes].
```

** To determine the PVC strategy, see xref:configuring-storage-strategies.adoc[]. 
+
.Preparing to run multiple workspaces on Amazon Elastic Block Store (Amazon EBS)

> ====

Reviewer: ```suggestion
IMPORTANT
====
```

Author: To me, IMPORTANT is meaningless here, it is just an example.

Reviewer: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions Is this an admonition block? If so, consider using one of the valid admonition types. If you don't want to use one of the valid admonition types, you can also remove the block markers and have the phrase as a single sentence paragraph.

Reviewer: It seems like an important piece of information that the user must know to choose the correct PVC strategy.

Author: It is an example, not an admonition block.

Reviewer: I see now how this looks like [example block syntax](https://docs.asciidoctor.org/asciidoc/latest/blocks/example-blocks/). But the content is not an example. It states a limitation and gives the user an instruction.

> link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] only supports the `ReadWriteOnce` access mode. Use the `per-workspace` or `unique` PVC strategy.

Reviewer: ```suggestion
link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] supports only the `ReadWriteOnce` access mode. Use the `per-workspace` or `unique` PVC strategy.
```

Reviewer: https://www.quickanddirtytips.com/education/grammar/%E2%80%9Conly%E2%80%9D-the-most-insidious-misplaced-modifier

* You have installed an instance of `{prod-short}` by using Helm and the default namespace `{prod-namespace}`.

> * The combination of PVC strategy and access mode meets the following criteria:

Author: ```suggestion
* The combination of PVC strategy and access mode matches one of the following cases:
```
